File: src/main/java/com/github/zly2006/reden/access/TickStageOwnerAccess.java
package com.github.zly2006.reden.access;

import com.github.zly2006.reden.debugger.TickStage;
import org.jetbrains.annotations.Nullable;

public interface TickStageOwnerAccess {
    @Nullable
    TickStage getTickStage$reden();
    void setTickStage$reden(@Nullable TickStage tickStage);
    boolean getTicked$reden();
    void setTicked$reden(boolean ticked);
}

File: src/main/java/com/github/zly2006/reden/access/TickStageOwnerAccess.kt
package com.github.zly2006.reden.access

import com.github.zly2006.reden.debugger.TickStage

interface TickStageOwnerAccess {
    @Suppress("INAPPLICABLE_JVM_NAME")
    @get:JvmName("getTickStage\$reden")
    @set:JvmName("setTickStage\$reden")
    var tickStage: TickStage?

    @Suppress("INAPPLICABLE_JVM_NAME")
    @get:JvmName("getTicked\$reden")
    @set:JvmName("setTicked\$reden")
    var ticked: Boolean
}
